src/openforms/validations/validators/formats.py

```diff
@@ -1,30 +1,29 @@
-from typing import TYPE_CHECKING, Optional, Protocol
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING
 
 from django.core.exceptions import ValidationError
 from django.utils.translation import gettext_lazy as _
 
 import phonenumbers
 from phonenumbers import NumberParseException
 
-from openforms.validations.registry import StringValueSerializer, register
+from openforms.validations.base import BasePlugin
+from openforms.validations.registry import register
 
 if TYPE_CHECKING:
     from phonenumbers.phonenumber import PhoneNumber
 
 
-class ParsePhoneNumber(Protocol):
-    def __call__(self, value: str) -> "PhoneNumber":
-        ...  # pragma: nocover
-
-
-class PhoneNumberBaseValidator:
-    country: Optional[str]
+class PhoneNumberBaseValidator(ABC):
+    country: str | None
     country_name: str = ""
     error_message = _("Not a valid %(country)s phone number")
-    value_serializer = StringValueSerializer
-    _parse_phonenumber: ParsePhoneNumber
 
-    def __call__(self, value, submission):
+    @abstractmethod
+    def _parse_phonenumber(self, value: str) -> "PhoneNumber":
```

Comment:
```suggestion
    def _parse_phonenumber(self, value: str) -> PhoneNumber:
```
combined with `from __future__ import annotations`?